ecommerce/views/v0/__init__.py

```diff
@@ -450,6 +470,19 @@ def decode_response(self, request):
         return Response({})
 
 
+class CheckoutInterstitialView(LoginRequiredMixin, TemplateView):
+    template_name = "checkout_interstitial.html"
+
+    def get(self, request):
+        checkout_payload = generate_checkout_payload(request)
```

Comment:
If the cart is empty, this will return a 406 response, and I think there are some other states that also return something other than a `dict`, so in these cases if the return value isn't a `dict`, we probably want to just return that value.

Ideally we'd redirect to a page with an error but for the sake of getting an MVP checkout flow merged, we'll skip that for now.